website/docs/d/ingress.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_ingress"
+sidebar_current: "docs-kubernetes-data-source-ingress"
+description: |-
+  Ingress is a collection of rules that allow inbound connections to reach the endpoints defined by a backend. An Ingress can be configured to give services externally-reachable urls, load balance traffic, terminate SSL, offer name based virtual hosting etc.
+---
+
+# kubernetes_ingress
+
+Ingress is a collection of rules that allow inbound connections to reach the endpoints defined by a backend. An Ingress can be configured to give services externally-reachable urls, load balance traffic, terminate SSL, offer name based virtual hosting etc.
+This data source allows you to pull data about such ingress, especially the loadbalancer dns.
```

Comment:
I like the first part of the sentence, but then it feels a little awkward toward the end. Maybe leave out the specifics about the loadbalancer, since there are examples below that show all the types of data that can be fetched.

```suggestion
This data source allows you to pull data about such ingress.
```